flake.nix

```diff
@@ -1,23 +1,54 @@
 {
   inputs = {
     # The nixpkgs channels we want to consume
-    nixpkgs-24_11.url = "github:NixOS/nixpkgs/nixos-24.11-small";
+    nixpkgs-24_05.url = "github:NixOS/nixpkgs/nixos-24.05";
+    nixpkgs-24_11.url = "github:NixOS/nixpkgs/nixos-24.11";
+    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
Maybe use the `-small` variants?

Reply:
Let's try the small when installing Forgio (and compare build/deploy time).